include/deal.II/fe/fe_dgq.h

```diff
@@ -435,8 +435,14 @@ class FE_DGQArbitraryNodes : public FE_DGQ<dim, spacedim>
 /**
  * Implementation of scalar, discontinuous tensor product elements based on
  * Legendre polynomials, described by the tensor product of the polynomial
- * space Polynomials::Legendre. As opposed to the basic FE_DGQ element, these
- * elements are not interpolatory and no support points are defined.
+ * space Polynomials::Legendre. The tensor product is achieved using
+ * TensorProductPolynomials and the ordering of shape functions, like in
+ * TensorProductPolynomials, is lexicographic. For instance, the ordering in 2d
+ * is $P_0(x)P_0(y),\ P_1(x)P_0(y),\ \ldots,\ P_n(x)P_0(y),\ P_0(x)P_1(y),\ 
```

Comment:
You have a trailing whitespace here:
```suggestion
 * is $P_0(x)P_0(y),\ P_1(x)P_0(y),\ \ldots,\ P_n(x)P_0(y),\ P_0(x)P_1(y),\
```

Reply:
Fixed in following commit.